app/assets/stylesheets/application.scss

```diff
@@ -320,81 +320,72 @@ body a:hover {
 }
 
 .bg {
-	position: fixed; 
-	top: 0; 
-	left: 0; 
+	position: fixed;
+	top: 0;
+	left: 0;
 	width: 100%;
-	  
+
 	/* Preserve aspet ratio */
 	min-width: 100%;
 	min-height: 100%;
 	left: 0px;
 	// top: 0px;
 	z-index: -1;
-	filter:brightness(50%);
-	
-  }
+	filter: brightness(50%);
+}
 
-  .center-buttons {
-	  text-align: center;
-  }
+.center-buttons {
+	text-align: center;
+}
 
-  #icons {
+#icons {
 	color: black;
-  }
-
-  #icons:hover {
-	  font-size: 28px;
-	  color: blue;
+}
 
-  }
+#icons:hover {
+	font-size: 28px;
+	color: blue;
+}
 
-  .title-formating {
-	  color: #fafeff;
-	  text-align: center;
-	  font-size: '5rem';
-  }
+.title-formating {
+	color: #fafeff;
+	text-align: center;
+	font-size: '5rem';
+}
 
-  .not-found{
+.not-found {
 	//   display: flex;
 	//   justify-content: center;
 	//   align-items: center;
-	  top: 150;
-	  left: 150;
-  }
+	top: 150;
```

Comment:
this seems to be missing units, might be worth checking if this actually is 150(px?)